app/models/member.rb

```diff
@@ -30,6 +30,14 @@ class Member < ApplicationRecord
     "Other",
   ].freeze
 
+  MARITAL_STATUSES = [
```

Comment:
should we validate that marital statuses are included in this list at the model level? I know we are doing so at the step level but might be good to have it here too so we have "normalized data" -- your call!

Reply:
yeah, I think that's a great idea! we decided not to for now because 1) we ended up not using marital statuses in our feature, and 2) it would likely introduce some validation problems with existing data when we deployed it. thought we could add validations for inclusion in a follow-up!